packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -132,56 +149,65 @@ const updateShippingMethod = async (
     });
 };
 
-export const resolver = config => {
+export const basketResolver = (config: Config) => {
     return {
         Query: {
-            getCart: async (_, {}, context) => {
-                const apiCart = await getBasket(config, context);
-                if (apiCart.fault) {
+            getBasket: async (_: never, {}, context: AppContext) => {
+                const apiBasket = await getBasket(config, context);
+                if (apiBasket.fault) {
                     logger.error(
-                        'ERROR Received when getting cart',
-                        apiCart.fault.message,
+                        'ERROR Received when getting basket',
+                        apiBasket.fault.message,
                     );
-                    throw new ApolloError(apiCart.fault.message);
+                    throw new ApolloError(apiBasket.fault.message);
                 } else {
-                    return new Cart(apiCart);
+                    return new Basket(apiBasket);
                 }
             },
         },
         Mutation: {
-            addProductToCart: async (_, { productId, quantity }, context) => {
-                let apiCart = await addProductToBasket(
+            addProductToBasket: async (
+                _: never,
+                parameters: { productId: string; quantity: number },
+                context: AppContext,
+            ) => {
+                const { productId, quantity } = parameters;
```

Comment:
This is a bit confusing, because a TypeScript object definition looks really similar to destructuring in JavaScript. Line 174 is destructuring the parameters, whereas line 171 is defining the types of those values.